src/inboxsdk.d.ts

```diff
@@ -339,6 +363,15 @@ export interface RowDescriptor {
   iconHtml?: string;
   /** An optional url to an icon to display on the left side of the row */
   iconUrl?: string;
+  /**
+   * Where to place the Row's icon.
+   *
+   * If unset, center the provided icon within the three icons at the start of the row. This was the behavior prior to the introduction of this property.
+   *
+   * If set to 'start', place the icon at the start of
+   * the three icons, collapse the two icons down, and add saved space to {@link RowDescriptor#title}'s width.
+   */
+  iconPosition?: 'start';
```

Comment:
Leaning toward axing this unless someone asks for it.

If we keep it, I'd rather the setting was named and described in a way emphasizing it was about keeping the icon and recipient columns of the row aligned with the icon and recipient columns of Gmail.

Reply:
ohh right makes sense, we just do it by default